content/news/XXXX-XX-XX-mixxx-2_5-released.md

```diff
@@ -0,0 +1,560 @@
+title: Mixxx 2.5 Release Announcement
+authors: Evelynne Veys
+tags: 2.5, release announcement
+comments: yes
+status: draft
+
+#### All I want for DJ-ing is ...
+
+Home alone? Scrooged? The sound of music? Die hard?
+Change plans: this year you will be having something better to do than watching old Christmas movies.
+Mixxx is now your secret "Santa comin' to town", our elves had "good hearts with the burning desire to work hard",
+to get you "Rockin' around the Christmas tree at the party hop" with "something mighty sweet to see".
+"For the present, look high, not low": Mixxx's "presents 'neath the tree"
+
+#### So this is the last goodbye ...
+
+Important: OS-Support.
+macOS versions earlier than 11 can have a "Last Christmas".
+Windows versions earlier than version 10 build 1809 can be "Driving home for Christmas".
+For Ubuntu versions earlier than 22.04 "Christmas Will Break Your Heart".
+By stepping over from QT5 to QT6.5 Mixxx has to respect the minimum system requirements stated by QT for the official builds.
+That's why the support has been dropped for macOS earlier than 11, Windows versions earlier then version 10 buikd 1889 and Ubuntu earlier then 22.04.
+
+#### Let's cheer for the elves
+
+Ho Ho Ho, our sleigh is filled with many shiny presents:
+
+* A red box filled with improvements to waveforms: 'SlipMode waveform' gives you visual aid
+while scratchin'
+
+@Video(https://www.youtube.com/watch?v=Xi3Ec0Jo-z8)
+
+* The 'beats until next marker' helps you never missing the break

```

Comment:
not a native speaker, but the previous version doesn't sound right (unless its supposed to be pun I don't get).
```suggestion
* The 'beats until next marker' helps you to never miss the break
```